### ISSUE TITLE
Jest 12.0.2 fails with Cannot read property '_jasmineAdapter' of undefined

### ISSUE THREAD

**Issue by timReynolds** [wallabyjs/public#589]
### Issue description or question
Updating to the latest version of Jest results in the following error message;

```
​​​​​TypeError: Cannot read property '_jasmineAdapter' of undefined
    at Runtime._execModule (C:\Source\WallabyTestSetupPF\PF\js\node_modules\jest-cli\src\Runtime\Runtime.js:318:17)
    at Runtime.requireModule (C:\Source\WallabyTestSetupPF\PF\js\node_modules\jest-cli\src\Runtime\Runtime.js:164:14)
    at C:\Source\WallabyTestSetupPF\PF\js\node_modules\jest-cli\node_modules\jest-jasmine2\src\index.js:68:20
    at FakeTimers.runWithRealTimers (C:\Source\WallabyTestSetupPF\PF\js\node_modules\jest-cli\node_modules\jest-util\lib\FakeTimers.js:260:7)
    at jasmine2 (C:\Source\WallabyTestSetupPF\PF\js\node_modules\jest-cli\node_modules\jest-jasmine2\src\index.js:36:26)
    at Test.run (C:\Source\WallabyTestSetupPF\PF\js\node_modules\jest-cli\src\Test.js:44:12)
    at C:\Source\WallabyTestSetupPF\PF\js\node_modules\jest-cli\src\TestRunner.js:340:68
```

### Wallaby.js configuration file

```javascript
var babel = require('babel');

module.exports = function (wallaby) {
    var wallabyBabelCompiler = wallaby.compilers.babel({
        babel: babel,
        stage: 0
    });

    return {
        debug: true,
        files: [
            { pattern: 'jest-setupEnvScriptFile.js', instrument: false },
            { pattern: 'jest-setupTestFrameworkScriptFile.js', instrument: false },
            'package.json',
            'src/**/*.js',
            '!src/**/__tests__/*.js'
        ],
        tests: [
            '__tests__/**/*.js',
            'src/**/__tests__/*.js'
        ],
        env: {
            type: 'node',
            runner: 'node',
            params: {
                runner: '--harmony'
            }
        },
        compilers: {
            'src/**/*.js*': wallabyBabelCompiler
        },
        workers: {
            initial: 10,
            regular: 5
        },
        testFramework: 'jest',
        bootstrap: function (wallaby) {
            var config = require('./package.json').jest;
            wallaby.testFramework.configure(config);
        }
    };
};
```

### Code editor or IDE name and version
Visual Studio 2015

### OS name and version
Windows

### Test Repository
[WallabyTestSetupPF](https://github.com/timReynolds/WallabyTestSetupPF)

**Comment by ArtemGovorov** [comment]
Thanks for the repo. How do I reproduce it, do I have to edit tests to get the error? If I just clone the repo, run `npm i` and run wallaby, it works for me.

**Comment by ArtemGovorov** [comment]
One more question, what's your NPM version (`npm -v`)?

**Comment by timReynolds** [comment]
NPM version is 3.6.0 but I just removed all my node modules and installed them from fresh which resolved the problem.

**Comment by SindreSvendby** [comment]
I had the same problem after switching from `ava` to `jest`, at first a clean install did not remove the problem, but then I realized I had not removed the `ava` dependency from `devDependencies` in `package.json`  problem was because `ava` was  still in the `node_modules` folder. After removing the dependency and then do a fresh `npm install` the problem was solved.  

So it looks like wallaby is doing something in the background and is not fully using the `testFramework` property, since it cares that a dependency is there, even if it should not be used. 

This issue helped me 99% on the way, hopefully this comment can help someone else
